### PR TITLE
Barrnap-0.4.2.eb (REVIEW)

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

File: easybuild/easyconfigs/b/Barrnap/Barrnap-0.4.2.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://github.com/Victorian-Bioinformatics-Consortium/barrnap/archive/']
+sources = ['%(version)s.tar.gz']
+
+# this is just a perl script but includes a "make test" option to verify it's working
+makeopts = ' test'
+
+# We require at least EasyBuild-1.12 to have glob support in the MakeCp easyblock
+easybuild_version = '1.12.0'
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/b/Barrnap/Barrnap-0.4.2.eb]: L23-L25, may be able to do without; @boegel; test/merge?

File: easybuild/easyconfigs/b/Barrnap/Barrnap-0.4.2.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ It supports bacteria (5S,23S,16S), archaea (5S,5.8S,23S,16S), mitochondria (12S,16S) 
+ and eukaryotes (5S,5.8S,28S,18S). """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://github.com/Victorian-Bioinformatics-Consortium/barrnap/archive/']
+sources = ['%(version)s.tar.gz']
+
+# this is just a perl script but includes a "make test" option to verify it's working
+makeopts = ' test'
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/b/Barrnap/Barrnap-0.4.2.eb]: buildopts

File: easybuild/easyconfigs/b/Barrnap/Barrnap-0.4.2.eb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+files_to_copy = ["*"]
+        
+sanity_check_paths = {
+    'files': ["bin/barrnap", "README.md"],
+    'dirs': ["binaries", "db", "examples"],
+}
+
+moduleclass = 'bio'
+
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/b/Barrnap/Barrnap-0.4.2.eb]: remove empty line..
lgtm , minor changes needed here,   put this in REVIEW

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/1eba7767db9b892bdacb for a full test report.

**Comment by boegel** [comment]
Depends on a Perl module being present:

```
== 2016-01-23 17:51:11,220 runpy.run DEBUG cmd " make -j 16  test" exited with exitcode 2 and output:
./bin/barrnap examples/small.fna
Can't locate Time/Piece.pm in @INC (@INC contains: /usr/local/lib64/perl5 /usr/local/share/perl5 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5 .) at ./bin/barrnap line 4.
BEGIN failed--compilation aborted at ./bin/barrnap line 4.
```

**Comment by boegel** [comment]
@pescobar ping on Perl module dep?

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**